spidermon/contrib/scrapy/monitors.py

```diff
@@ -38,7 +38,10 @@ class BaseStatMonitor(BaseScrapyMonitor):
 
     Create a monitor class inheriting from this class to have a custom
     monitor that validates numerical stats from your job execution
-    against a configurable threshold.
+    against a configurable threshold. If this threshold is set in Scrapy
```

Comment:
I would replace mention to Scrpay Cloud to something else more general like passing settings from the CLI